### PR TITLE
Fix overriding webpack loaders example

### PR DESCRIPTION
**Pull request by vivcogit**
Issue: none

## What I did
Fix example to actual structure webpack's config

## How to test
Create a storybook with custom webpack config like at example

### CODE REVIEW

File: docs/src/pages/configurations/custom-webpack-config/index.md [hunk 1]
```diff
@@ -253,6 +253,8 @@ const path = require('path');
 const custom = require('../webpack.config.js');
 
 module.exports = async ({ config, mode }) => {
-  return { ...config, loaders: custom.loaders };
+  config.module.rules.push(...custom.module.rules);
```
Comment (shilman) [inline docs/src/pages/configurations/custom-webpack-config/index.md]: Thanks for the PR! However, I don't think this is correct.

The old example replaces the loaders with your app's loaders.

Your change appends the loaders to the default loaders provided by storybook.

I think the intent of the example is to use your existing webpack config, which means replacing Storybook's loaders entirely.

Let me know what you think!

cc @ndelangen @igor-dv

**Review by shilman — CHANGES_REQUESTED** [review]
See my comment above ⬆️

File: docs/src/pages/configurations/custom-webpack-config/index.md (hunk 1 above)
Comment (vivcogit) [reply docs/src/pages/configurations/custom-webpack-config/index.md]: Thanks for Review! I agree with you and update PR soon.

Comment (vivcogit) [reply docs/src/pages/configurations/custom-webpack-config/index.md]: I updated a PR, review again please.

Comment (shilman) [reply docs/src/pages/configurations/custom-webpack-config/index.md]: Aha I didn't see the bug in the original. Good catch and thanks for following up on this! ❤️

**Review by shilman — APPROVED** [review]

**Pull request merged**